sys/cheri/cheri_syscall.c

```diff
@@ -40,5 +40,10 @@
 int
 cheri_syscall_authorize(struct thread *td)
 {
+#ifdef __aarch64__
+	/* XXX Dapeng: Remove when c18n is ported to CHERI-RISC-V */
```

Comment:
I suppose you could instead hoist the sysctl out of the ifdef below since this isn't necessarily compartmentalization specific.

Reply:
Indeed. Fix pushed now.